### PR TITLE
[feature]modify err log

### PR DESCRIPTION
**Pull request by yunfeiyanggzq**
Signed-off-by: yunfeiyangbuaa <yunfeiyang@buaa.edu.cn>

#### Type of change

- Bug fix
- New feature
- Improvement (improvement to code, performance, etc)
- Test update
- Documentation update

#### Description

the error log isn't clear.I put out log with more information

### CODE REVIEW

**Comment by yunfeiyanggzq** [comment]
sorry,I  will do the cli check later

File: lib/server/idemix/idemixlib.go [hunk 1]
```diff
@@ -37,7 +37,7 @@ func (i *libImpl) GetRand() (rand *amcl.RAND, err error) {
 	defer func() {
 		r := recover()
 		if r != nil {
-			err = errors.Errorf("failure: %s", r)
+			err = errors.Errorf("Failed to get rand: %s", r)
```
Comment (btl5037) [inline lib/server/idemix/idemixlib.go]: Error strings should not start with a capital letter unless it is a proper noun or an acronym

Comment (yunfeiyanggzq) [reply lib/server/idemix/idemixlib.go]: thanks for your review,I will  fix this it.

**Comment by sykesm** [comment]
@yunfeiyanggzq - Are you planning to address @btl5037's comments soon?

Thanks.

**Comment by sykesm** [comment]
@yunfeiyanggzq, thank you for the contribution but it's been a while since we've heard from you.

I'm going to close this PR. If you find time to come back to it and update it to address the comments you've received and to make the tests pass, just reopen the PR and we'll take another look.

Thanks.

**Pull request closed without merging**